libmamba/src/core/output.cpp

```diff
@@ -146,7 +150,7 @@ namespace mamba
                             out,
                             "{: >{}}",
                             fmt::styled(row[j].s, row[j].style),
-                            cell_sizes[j] + m_padding[j]
+                            cell_sizes[j] + static_cast<std::size_t>(m_padding[j])
```

Comment:
Should `m_padding` `valut_type` be unsigned?

Reply:
Yes, but this i sa data member